README.md

```diff
@@ -1311,6 +1311,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Nur Akmal Mohd Nordi](https://github.com/nurakmal920207)
   - [Nwoke Emmanuella](https://github.com/Ellah120)
   - [Nworie Chikwado Emmanuel](https://github.com/chikwado17)
+  - [Nkanmuo Vera Chioma](https://github.com/chiomavera)
```

Comment:
Please add your name in line 1306, after Nitish Kumar.
Let us know if you need any help with that. 🙂

Reply:
Okay I will do that, thanks.

Reply:
@Panquesito7 
I have made the necessary changes as you requested